week1/project_config.yml

```
@@ -0,0 +1,32 @@
+  - HIV_AIDS
+  - GDP
+  - Population
+  - thinness_1-19_years
+  - thinness_5-9_years
+  - Income_composition_of_resources
+  - Schooling
+
+cat_features:
+  - Status
+
+target: Life_expectancy
```

yamllint

[error] no new line character at the end of file

(new-line-at-end-of-file)